### DISCUSSION TITLE
Menu Icon Mod for VWG11

### DISCUSSION THREAD

**Discussion by djskual** [Mr-MIBonk/M.I.B._More-Incredible-Bash#289] [Ideas]
Hello,

Possible to create an menu icon mod for VWG11 like for already done VWG13.
Or something similar to have more modern icons on the MIB2 menu?

**Comment by harman-f** [comment]
If you/someone would prepare this skin, yes. 

Just nothing public/shared so far.

**Reply by djskual** [reply]
off course, i think i can prepare it, but i need someone to explain to me the process...

and other thing, i seen some mib2 high vwg11 with 8 icons on the same menu screen. so you thing its possible to change this too?

**Reply by djskual** [reply]
ok, i've dumped my skins. then extracted images from skin 1.
Now i think i need to replace good ones, then recompress... but i found same images in different sizes... i'm a little bit confused...
Help please.

**Reply by djskual** [reply]
think i've found something to switch the menu in grid mode.

in info.txt, do you think if i change the line "DefaultMenuMode=FlowList" to "Grid", that will do the job?
or need i to change something else... perhaps the name of the skin.

Then, if i want to test if that work, what i need to do to have another skin to select? (skin5)

**Reply by djskual** [reply]
can someone could send to me the skins dump from a VWG13 unit with modified icons skins? that will help me

**Reply by harman-f** [reply]
Can send you link later tonight. 
you will also have to resize the images to fit G11 screen. They need to hav e a specific seize and stuff. And do it with a proper program, so that there are minimal Artefakts.

**Reply by djskual** [reply]
Yes, i want link please. I use photoshop to resize.
What do you think about the grid layout?

**Reply by djskual** [reply]
i can't get my linux to compress the images.mcf file
don't understand

**Reply by djskual** [reply]
ok, found that compress utility need python2
but now, i get an error about missing mod P in the image i modified
an idea?

**Reply by djskual** [reply]
OK, i finally got this to work. i used a program to compress my png but can't compress the images.mcf file.
i've done a test with the OPS menu in skin0
now, i have a blue guide line instead of orange one.

![20230726_200809](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/4947091/1c32ba93-999b-43d0-80b8-25978a952821)

**Reply by harman-f** [reply]
Check MIB2TOOLBOX on git for tools to handle mcf files

**Reply by djskual** [reply]
i decompressed and compressed files with mib2toolbox utilities.

Now i need g13 modified dump file to work on menu icons

**Reply by harman-f** [reply]
Just trying to find the "original" Alpine Skin base....
There have been many adaptions for various MIB2 models already.

Some findings on the Alpine adaption for VWG13:
https://mibwiki.one/s/9d77d97f-4deb-4ac7-a28d-1752eb6c0211

There are some more nice skins, which still have to find their way into M.I.B:
For some reason they are all G13/MIB2.5 based.
![photo_2023-06-27_08-30-10](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/98130152/3f73a96f-5ecb-4e1c-90c1-dd55290f9f4f)
![photo_2023-06-20_22-25-18](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/98130152/8fd063b8-852f-4ca3-9e1d-c5397fdd888e)
![photo_2023-06-25_09-21-07](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/98130152/308f8ed4-443b-4703-89a0-7c83f79da350)

**Reply by harman-f** [reply]
[ALPINE_VWG13.zip](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/12177699/ALPINE_VWG13.zip)
These are already converted for VG13. Will try to get you the original files to start with...

**Reply by djskual** [reply]
Thanks a lot for those informations and files. not sure to have lot of time to work on this, but step by step...

other question, what do you think about the grid layout on G11? i seen this on some facelift G11. but can't test as mib2toolbox only change images and not the info.txt file

**Reply by harman-f** [reply]
Latest FWs should have the option in settings to switch to grid menu.

If your car is WLAN reach, you could install SSH via MIB and connect via SSH, which will also enable file transfer (MobaXTerm)

**Reply by djskual** [reply]
thnak you for the ssh tip, but not sure to well understand how that work.

in the first time, i will just change images, then i will see...

to include the skin in M.I.B., please tell me exatly which files and which format you need.
sure you don't want my changes in skin0.

let me know.

**Reply by djskual** [reply]
i've found a function in toolbox to activate a setting to switch between horizontal layout and grid.
now, i can test the skin in grid.

**Comment by djskual** [comment]
Ok, i've made a GTE pre-facelift pack for skin0.
I've changed all car's pictures to match GTE style.
Are you interested?

Exemple before/after:
![img_1243](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/4947091/8450d4f6-034f-4d8d-a237-2c061e5b0f45)
![img_1243](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/4947091/bab902ec-867f-46de-b172-edca3c7b716d)

Installed in my MIB (before/after):
![avant](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/4947091/eaab5341-b19a-438e-bad9-f327a751a8e3)
![apres](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/4947091/75525901-c21f-45ce-b846-f346b4265976)

**Reply by mkopsidas** [reply]
how did you installed in MIB ?
Thanks

**Reply by djskual** [reply]
> how did you installed in MIB ? Thanks

Hello, i installed this with M.I.B or mib2-toolbox... sorry don't remember

**Comment by djskual** [comment]
hello, could someone to send to me image.mcf files extracted from skin0 and skin1 from latest VWG13 firmware??
Thanks a lot for the help.

**Comment by djskual** [comment]
Hello, The skin is ready and working on my mib.
What files do you need?
do you whant the gte skin0 too?
Let me know.

![20230805_085118](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/4947091/751d5fac-8cae-4d32-ac79-d891bae98205)

![20230808_065112](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/4947091/8d153a30-ecb7-4699-9443-1743f8fe1a3c)

**Reply by djskual** [reply]
up!!!

**Comment by Georgegr783** [comment]
Hello . i need help i downloaded v1.3.3 for a tecnisat discover. I have car play activated and everything went well. but i tried to change skins and the folder was empty. i downloaded the mib std2 skins customization and after unzipping it i got the 2 folders that were full. I placed them in the sd in the respective numbers. I put the card in the device I went to the green screen tools custom I make copy skins and it says no new skins exist. Can someone help me? what am I doing wrong now is the correct procedure?

**Comment by JLAM-MIB** [comment]
Hello.
Does anyone have a Carbon Fiber or GTI Full Red conversion kit for a VW G13 K4525?

I recently bought a GTI MK7.5 and I'm fascinated by what can be done with my MIB2.5 VW Discover Pro - MHI2_ER_VWG13_K4525_1_AIO_MU1367_20230318 unit.
